src/main/java/user/createCorporatePage/PageGenerateController.java

```diff
@@ -0,0 +1,48 @@
+package user.createCorporatePage;
+
+
+import entity.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class PageGenerateController {
+    final IPageGenerateInput interactor;
+
+    @Autowired
+    public PageGenerateController(IPageGenerateInput interactor) {
+        this.interactor = interactor;
+    }
+
+    @PostMapping
+    public PageGenerateResponseModel create(PageGenerateRequestModel requestModel){
+        return interactor.create(requestModel);
+    }
+
+
+
+//    final PageGenerateInputBoundary pageInput;
```

Comment:
As you said in lab, this is code that is not required anymore but you want to save here just in case. You should explain that in a comment above this commented out area so that people don't think this is unfinished code.

Reply:
addressed